activemodel/lib/active_model/dirty.rb

```diff
@@ -189,7 +189,7 @@ def attribute_was(attr) # :nodoc:
     end
 
     # Handles <tt>*_previously_changed?</tt> for +method_missing+.
-    def attribute_previously_changed?(attr, options = {}) #:nodoc:
+    def attribute_previously_changed?(attr) #:nodoc:
```

Comment:
What's the story re. changing the method signature? Is it related to this PR or just seem superfluous? Is it used/overridden/expected elsewhere?